exec/java-exec/src/main/java/org/apache/drill/exec/physical/impl/join/MergeJoinBatch.java

```diff
@@ -403,14 +325,14 @@ private JoinWorker generateNewWorker() throws ClassTransformationException, IOEx
           outputType = inputType;
         }
         JVar vvIn = cg.declareVectorValueSetupAndMember("incomingLeft",
-                                                        new TypedFieldId(inputType, vectorId));
+          new TypedFieldId(inputType, vectorId));
```

Comment:
Can you look if we can factor this out? I believe we have a utility method that builds vector to vector copies rather than by doing it manually.